playbook/app/pb_kits/playbook/pb_lightbox/_lightbox.tsx

```diff
@@ -37,15 +38,19 @@ const Lightbox = (props: LightboxType): React.ReactNode => {
     id = '',
     initialPhoto = 0,
     photos,
+    onChangeIndex = noop,
```

Comment:
1) I think we should still call this onChange, and not onChangeIndex because it's generic enough of an event to just be called using onChange. And eventually, we might want to pass in multiple parameters. 
Any reason to be more specific with onChangeIndex?

2) Since I have a story on the backlog for removing lodash, let's not add that `noop` as a dependency here. 
Instead can you try what I have below in my suggested commit and see if that works. That way, we can avoid one more spot i'll have to remove when i remove lodash.

Reply:
Makes sense! I'll make the changes and make another commit. Thank you for the feedback!